beacon-chain/rpc/prysm/v1alpha1/beacon/validators.go

```diff
@@ -528,7 +528,7 @@ func (bs *Server) GetValidatorParticipation(
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "Could not pre compute attestations: %v", err)
 		}
-	case version.Altair, version.Bellatrix:
+	case version.Altair, version.Bellatrix, version.Capella:
```

Comment:
Do you think instead of switch we can just do 
```
if stateversion >= version.Altair
```
That way we dont have to do this again in the next fork